CMakeLists.txt

```diff
@@ -1,10 +1,11 @@
 cmake_minimum_required(VERSION 3.5)
 project(slam_publisher)
 
+# Add the FastSLAM directory
 add_subdirectory(FastSLAM)
+add_library(LidarSubscriberLib src/lidar_subscriber.cpp)
```

Comment:
I don't know if it's best practice to add a single source file that's meant to be run to a separate library. For now we can keep it, but this should be refactored in the future.

Reply:
I totally agree. I was really struggling to properly link the header and source files and tried a bunch of things. For some reason that was the only thing that worked so I left it